about.html

```diff
@@ -0,0 +1,142 @@
+                  <span>ted@gmail.com</span>
+                </div>
+              </div>
+            </section>
+              
+              <section>
+              <div class="mid">
+                  <div class="mid-text">
+                    <h4>Creative Commons Global 2023 Logo</h4>
+                    <hr class="hr1">
+                    <p class="mid-p">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.<br/>After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.<br/>After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+                    </p>
+                  </div>
+                  <div class="mid-image">
+                    <img src="./Images/143849_ted_icon.png" alt="concert logo" class="img-mid">
+                  </div>
+                </div>
+              </section>
+              
+              <section>
+                <div class="past">
+                  <div class="past-text">
+                    <h3 class="past-h3"> See the past CC Global Summits</h3>
+                    <hr class="past-hr">
+                    <p class="past-p"> Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+                  </div>
+                  <div class="collage-exhibit-section">
+                    <div class="exhibit-img">
+                      <div class="exhibit-inner">
+                        <h3>2022</h3>
+                        <p class="img-p">TED 2022 Event that Inspiring in US</p>
+                      </div>
+                    </div>
+                    <div class="exhibit-img">
+                      <div class="exhibit-inner">
+                        <h3>2021</h3>
+                        <p class="img-p">TED 2021 Event that Inspiring in US</p>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </section>
+              
+              <section>
+              <div class="sponsors">
+                  <p>Partner</p>
+                  <hr>
+                  <div class="spons">
+                    <img src="./Images/mozilla 1.png" alt="mozilla_logo">
+                    <img src="./Images/googlekorea 1.png" alt="google_logo">
+                    <img src="./Images/naver 1.png" alt="naver_logo">
+                    <img src="./Images/airbnb 1.png" alt="airbnb_logo">
+                    <img src="./Images/daumkakaologo 1.png" alt="airbnb_logo">
+                  </div>   
+                </div>
+              </section>
+              
+                <footer>
+                  <img class="footer-img" src="./Images/143849_ted_icon.png" alt="Henna-design-logo">
+                  <p class="footer-p">2023 TED Event 
+                      All Rights Reserved</p>
+                </footer>
```

Comment:
- [ ] [OPTIONAL] Well done job but there's an overflow on the `y-axis` when on the desktop view as indicated below. Could fix that to improve your design and user experience?

![image](https://github.com/newtonah0/Microverse-Capstone-1/assets/99715640/c4052ca9-d04b-4501-8bad-517f9b7fcc65)